### PR TITLE
push down limit to mysql external table

### PR DESCRIPTION
**Pull request by Pslydhh**
- push down limit operator to mysql external table.

### CODE REVIEW

File: gensrc/thrift/PlanNodes.thrift [hunk 1]
```diff
@@ -237,6 +237,7 @@ struct TMySQLScanNode {
   2: required string table_name
   3: required list<string> columns
   4: required list<string> filters
+  5: required i64 limit
```
Comment (kangkaisen) [inline gensrc/thrift/PlanNodes.thrift:240]:
```suggestion
  5: optional i64 limit
```

File: be/src/exec/vectorized/mysql_scan_node.cpp [hunk 2]
```diff
@@ -27,6 +27,7 @@ MysqlScanNode::MysqlScanNode(ObjectPool* pool, const TPlanNode& tnode, const Des
           _tuple_id(tnode.mysql_scan_node.tuple_id),
           _columns(tnode.mysql_scan_node.columns),
           _filters(tnode.mysql_scan_node.filters),
+          _limit(tnode.mysql_scan_node.limit),
```
Comment (kangkaisen) [inline be/src/exec/vectorized/mysql_scan_node.cpp:30]: Need check this limit whether set.

**Review by kangkaisen — APPROVED** [review]

**Comment by Pslydhh** [comment]
run starrocks_fe_unittest

**Review by Seaven — APPROVED** [review]

**Comment by Seaven** [comment]
run starrocks_fe_unittest

**Pull request merged**